_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+This is to Kubernetes as what the Docker CLI is to Docker. Kubernetes 
+can also be managed through a GUI control panel which, as much as Kubectl does, 
+communicates with the Kubernetes Clusters through the API endpoints.
+
+
+__Minikube__
+
+Kubernetes can be run on any container cluster, including  custom ones built by 
+yourself. However, _probably_, the most popular destination is the Google Cloud 
+Platform (GCP) followed by Amazon Web Services (AWS). In any case, during 
+development, testing or learning it is best to do so locally, at least at 
+first. Minikube does exactly this. It mimics an orchestration platform within 
+your local workstation. Allowing us to deploy all our containers in their 
+respective Nodes locally as if they were hosted on the Cloud. This is great 
+while learning and testing new setups but also indispensable when testing a 
+production scenario without having to lease any external services on the 
+cloud until you're ready to do so, fully knowing what will be needed and when.
+
+__Contexts__
+
+So, we use Kubectl to manage our local Minikube cluster and maybe a few 
+more clusters on the cloud. These different orchestration targets are contexts 
+which we can tell Kubectl to switch between.
+
+
+
+## Deploying the Laravel Application to Minikube
+
+The previous section, explaining some of the terms used when working with 
+Kubernetes may intimidate most of us, but let's not be. These are 
+simply words to compartmentalise different layers and attributes within a 
+deployment scenario.
+ 
+I have kept this example quite simple by removing state, i.e. no persistence 
+and thus no databases. In this example, we will use Minikube to locally deploy
+ a basic Laravel application across three nodes for load balancing.
+ 
+_Want to learn how to manage storage volumes and databases? We will be 
+writing a tutorial about this soon. Subscribe and be notified as
+ soon as new articles are published._
+ 
+__The application__
+
+I have provided a simple Laravel application which can be cloned from <a href="https://github.com/learnk8s/laravel-kubernetes-demo" target="_blank" rel="noopener" class="link">GitHub</a>. This is nothing more than a new 
+Laravel installation. Therefore you can follow this example using either the 
+demo application or simply create your own new Laravel application. To use 
+the demo application simply clone in your project's directory.
+
+```bash
+cd /to/your/working/directory
+git clone git@github.com:learnk8s/laravel-kubernetes-demo.git .
+```
+
+__Prerequisites__
+
+In order to follow with this demonstration, you will need the following 
+installed on your local system:
+
+1) <a href="https://docs.docker.com/install/" target="_blank" 
+rel="noopener">Docker</a>
+
+2) <a href="https://kubernetes.io/docs/tasks/tools/install-kubectl/" target="_blank" 
+rel="noopener">Kubectl</a>
+
+3) <a href="https://github.com/kubernetes/minikube/releases" target="_blank" 
+   rel="noopener">Minikube</a>
```

Comment:
Why not using the standard syntax for markdown links?

Reply:
I copied your way of doing this for the external links from the about-us page. AFAIK there's no way to add both the "target" and "rel" attributes using the standard MD syntax. Let me know if you have another way.